dir.traversal.targets

```diff
@@ -141,4 +141,6 @@
   <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
   <Target Name="RebuildAndTest" DependsOnTargets="Rebuild;Test" />
   <Target Name="Test" />
+  <Target Name="Restore" />
```

Comment:
Can you please explain why this is necessary? Didn't we already override the restore mechanism?

Reply:
With Arcade there are some common targets that get called on root level projects and a number of our root level projects like src/dir.proj for example don't need a restore but we need to the target defined to enable the scenario. For projects that do want to do something during restore like build.proj we override this Restore target.

Reply:
Got it, thanks.